packages/muelu/doc/Tutorial/index.rst

```diff
@@ -0,0 +1,96 @@
+.. First_Steps documentation master file, created by
+   sphinx-quickstart on Wed Mar 18 00:30:46 2020.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
+
+==================
+The MueLu Tutorial
+==================
+
+.. epigraph::
+
+   This is the MueLu Tutorial.
+   Additional resources can be found at the `MueLu package web page <https://trilinos.github.io/muelu.html>`_,
+   in the `MueLu User's Guide <https://github.com/trilinos/Trilinos/tree/master/packages/muelu/doc/UsersGuide>`_
+   and the `Doxygen source code documentation <https://trilinos.org/docs/dev/packages/muelu/doc/html/index.html>`_.
+
+Preface
+=======
+
+The MueLu tutorial is written as a hands-on tutorial for MueLu,
+the next generation multigrid framework in Trilinos.
+It covers the whole spectrum from absolute beginners' topics to expert level.
+Since the focus of this tutorial is on practical and technical aspects
+of multigrid methods in general and MueLu in particular,
+the reader should already have a basic understanding of multigrid methods and their general underlying concepts.
+Please refer to multigrid textbooks for the theoretical background.
+
+Content
+=======
+
+The tutorial is split into three parts.
+The first part contains four tutorials for beginners who are interested in using multigrid methods.
```

Comment:
Not stating a number here makes us future-proof in case of adding more tutorials...
```suggestion
The first part contains tutorials for beginners who are interested in using multigrid methods.
```